### PR TITLE
Svelte 1.4

### PR DESCRIPTION
**Pull request by Bloctans**
image
(my maybe, beta 6 and for tnix to fix
image
all of the other stuff to do (might do in another branch)

this adds:
image

### CODE REVIEW

File: src/lib/Dialog.svelte
Comment (JoshAtticus) [inline src/lib/Dialog.svelte:1]: Why was this file deleted?

**Review by JoshAtticus — APPROVED** [review]
Seems fine to me, I'm not too experienced, so you might want CST's approval before merging.

File: src/lib/Dialog.svelte
Comment (Bloctans) [reply src/lib/Dialog.svelte:1]: dialogs were replaced with modals

**Comment by Bloctans** [comment]
> Seems fine to me, I'm not too experienced, so you might want CST's approval before merging.

requested tnix approval and cst approval

**Comment by JoshAtticus** [comment]
> image

You can probably add image uploading with go.meower.org, you just need a button.

**Comment by Bloctans** [comment]
> > image
> 
> You can probably add image uploading with go.meower.org, you just need a button.

would, but idk how, might do in beta 6 or my next svelte branch

**Comment by JoshAtticus** [comment]
> > > image
> > 
> > 
> > You can probably add image uploading with go.meower.org, you just need a button.
> 
> would, but idk how, might do in beta 6 or my next svelte branch

If tnix didn't delete his revolt account, I could've gotten the documentation he sent me.

**Comment by Bloctans** [comment]
> > > > image
> > > 
> > > 
> > > You can probably add image uploading with go.meower.org, you just need a button.
> > 
> > 
> > would, but idk how, might do in beta 6 or my next svelte branch
> 
> If tnix didn't delete his revolt account, I could've gotten the documentation he sent me.

oof

**Comment by Bloctans** [comment]
what

**Comment by tnix100** [comment]
> 

go.meower.org is a link shortner, not a CDN, there will be a proper uploads server in b6 that will handle file uploads.

**Comment by JoshAtticus** [comment]
> > 
> 
> go.meower.org is a link shortner, not a CDN, there will be a proper uploads server in b6 that will handle file uploads.

I can host a file upload server for b6.

**Comment by Bloctans** [comment]
> > > 
> > 
> > 
> > go.meower.org is a link shortner, not a CDN, there will be a proper uploads server in b6 that will handle file uploads.
> 
> I can host a file upload server for b6.

please

**Comment by tnix100** [comment]
> > > 
> > 
> > 
> > go.meower.org is a link shortner, not a CDN, there will be a proper uploads server in b6 that will handle file uploads.
> 
> I can host a file upload server for b6.

It's gonna be hosted on the same server as the API, and our image proxy will be on Azure. We are not dealing with multiple server scaling until probably after release.

**Review by ArrowAced — APPROVED** [review]

**Comment by CST1229** [comment]
@Bloctans  @tnix100 (re)requesting reviews because i made changes

**Comment by Bloctans** [comment]
> @Bloctans @tnix100 (re)requesting reviews because i made changes

can it be merged after that?

**Comment by tnix100** [comment]
I'll add typing indicators then I think it can be merged

**Review by Bloctans — COMMENTED** [review]
"Pull req authors cant approve thier own commit"

seems fine

**Comment by CST1229** [comment]
> can it be merged after that?

yes

**Comment by tnix100** [comment]
@CST1229 @Bloctans I made more changes, please review

**Comment by CST1229** [comment]
> @CST1229 @Bloctans I made more changes, please review

lgtm

**Comment by tnix100** [comment]
@CST1229 @Bloctans Typing indicators, quotes, and line breaks done. Please review, and then hopefully we can merge as 1.4.

**Review by tnix100 — APPROVED** [review]

**Review by Bloctans — COMMENTED** [review]
approved

**Comment by Bloctans** [comment]
approved everything

**Review by CST1229 — APPROVED** [review]
though the disconnect modal while logging out might need a fix

**Comment by tnix100** [comment]
> though the disconnect modal while logging out might need a fix

I did notice that during development, only happened a few times, if you can find a fix to it that'd be great

**Comment by Bloctans** [comment]
> > though the disconnect modal while logging out might need a fix
> 
> I did notice that during development, only happened a few times, if you can find a fix to it that'd be great

oof, i think thats because i was playing around with the code and forgot to do that

**Comment by Bloctans** [comment]
Push! Push! Push! Push! Push! Push!

**Comment by tnix100** [comment]
Push time?????

**Comment by Bloctans** [comment]
> Push time?????

yes push time please even tho cst hasnt approved

**Comment by CST1229** [comment]
> > Push time?????
> 
> yes push time please even tho cst hasnt approved

i have


(in other words: you can merge)

**Comment by showierdata9978** [comment]
@tnix100

**Comment by tnix100** [comment]
Merged!

**Comment by Bloctans** [comment]
commit descriptions (inital dev commits):
yes - create group chats and start on member list
h - weird change, i dont know why.
Fix spam by holding down enter while ratelimited - the name of the commit + oops i need to fix 
chat list almost done - sorta finish up the chat member feature
Add files via upload - add modals feature to my branch (also fix and add some small things)
Unused - delete dialog.svelte
Add files via upload - fix modal buttons
'gc member stuff part 1 + uh' - add bridged tag and group chat member profile preview modal
yes 2 - has a changelog in disc
add and remove member part 1 - leave chat and add members
leavechat time - the name
final changes - tab circles in posts and remove red bg from chat members

**Comment by Bloctans** [comment]
commit descriptions (outside fixes):
fix bugs - fix replying to Notification and Announcement and fix viewing profile of server
will continue tomorrow

**Comment by mdwalters** [comment]
> Merged!

ok

**Pull request merged**
